### ISSUE TITLE
Can we have a way to reset a scheduler back to epoch -1

### ISSUE THREAD

**Issue by DrAlta** [pytorch/pytorch#41623]
## Motivation
I'm tring to calculate the minimun loss of a NN.

## Pitch
something like .step(-1) but that's going away as deprecated.

## Alternatives

Make a new scheduler.

**Comment by vincentqb** [comment]
Can you give an code example of what you are trying to do with a learning rate scheduler? Are you trying to get the initial learning rate via `scheduler.step(-1)`?
